examples/tfengine/modules/team.hcl

```diff
@@ -490,10 +490,30 @@ template "additional_iam_members" {
 
 # Kubernetes Terraform deployment. This should be deployed after the GKE Cluster has been deployed.
 template "kubernetes" {
-  recipe_path = "{{.recipes}}/deployment.hcl"
+  recipe_path = "{{.recipes}}/project.hcl"
   output_path = "./kubernetes"
 
   data = {
+    project = {
+      project_id = "{{.prefix}}-{{.env}}-apps"
+      exists     = true
+    }
+    resources = {
+      kubernetes_service_accounts = [{
+        name = "ksa-gke"
+        namespace = "namespace"
+        google_service_account_email = "runner@{{.prefix}}-{{.env}}-apps.iam.gserviceaccount.com"
```

Comment:
Does this GSA get created by the Workload Identity CFT module?

Reply:
No, we require users to create the GSA using the `service_accounts` resrouce.